apps/publish/enqueue/enqueue_service.py

```diff
@@ -475,19 +475,16 @@ def queue_transmission(
             subscriber_codes = {}
 
         try:
-            if config.PUBLISH_ASSOCIATIONS_RESEND and not publish_first_version:
-                if (
-                    config.PUBLISH_ASSOCIATIONS_RESEND == "default"
-                    and not doc.get("rewrite_of")
-                    and doc.get("state") not in ["corrected", "being_corrected"]
-                ):
+            if config.PUBLISH_ASSOCIATIONS_RESEND and not new_item:
```

Comment:
with that I'm not sure how does it resend for new items?

Reply:
Yes, it will resend a new item also but I think we should keep its name `publish_first_version` as we discussed earlier https://github.com/superdesk/superdesk-core/pull/2173#discussion_r613023888 so that it will not make any confusion.

Reply:
where does it resend new items?

Reply:
`new_item` is just a bool variable to prevent from resending the same association multiple times and as per the current behavior `media items should just be sent once (current behavior)` which means we are already resending media association for new items.

Reply:
so it's like if we have same file attached multiple times?

Reply:
No, Suppose we have a **new story with only one association** and as per the `current behavior` it is already resending the media association means we have 2 entries in the publishing queue. after that, if config PUBLISH_ASSOCIATIONS_RESEND is set then from the main story we are fetching association and resending again then we will have one more entry in the publishing queue. so total we have 3 entries(main story, association, again association) instead of 2(main story, association).so prevent extra entries I have added few checks.

Reply:
I see but does it resend for new items if the associations was already published before?

Reply:
yes, it will resend. I have cross-checked it.

Reply:
ok, so maybe calling that param `sent` would be better than, I thought it's about new item being published